modin/tests/pandas/dataframe/test_default.py

```diff
@@ -1485,3 +1485,16 @@ def test_setattr_axes():
 def test_attrs(data):
     modin_df, pandas_df = create_test_dfs(data)
     eval_general(modin_df, pandas_df, lambda df: df.attrs)
+
+
+def test_df_from_series_with_tuple_name():
+    # Tests that creating a DataFrame from a series with a tuple name results in
+    # a DataFrame with MultiIndex columns.
+    pd.testing.assert_index_equal(
```

Comment:
Hi @noloerino! Overall it looks good! However, it may be more reliable to check the entire objects themselves, and not just the index (for example, using `eval_general`). What do you think?

Reply:
Sounds good. I didn't add `eval_general` since this is testing the DF constructor and not any methods on top of it, but I changed it to use `df_equals` instead.